ee/hogai/tools/create_insight.py

```diff
@@ -16,7 +16,7 @@
 from ee.hogai.utils.types.base import AssistantNodeName, AssistantState, NodePath
 
 INSIGHT_TOOL_PROMPT = """
-Use this tool to generate an insight from a structured plan. It will return a visualization that the user will be able to analyze and textual representation for your analysis.
+Use this tool to generate an insight from a structured plan. It will return a visualization that the user can analyze and a textual representation for your analysis. This tool can be used to create new insights or to edit existing insights. To edit an existing insight, you need to generate a new one based on the schema of the existing insight.
```

Comment:
A bit more readable:

```suggestion
Use this tool to generate an insight from a structured plan. It will return a visualization that the user can analyze and a textual representation for your analysis. This tool can be used to create new insights or to edit existing insights. To edit an existing insight, use this tool with a new plan based on the schema of the existing insight.
```